src/seedu/addressbook/parser/Parser.java

```diff
@@ -135,6 +129,13 @@ private Command prepareAdd(String args) {
         }
     }
 
+    private Command prepareSort(String args) {
+
```

Comment:
[Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html) violation! One more more of these problems detected:
1. extra/missing spaces.
2. brace style problem.
3. missing braces.